### ISSUE TITLE
Tests missing from PyPI

### ISSUE THREAD

**Issue by jtojnar** [jtauber/pyuca#21]
I am trying to package pyuca for NixOS but cannot run tests because the PyPI package lacks them.

See also the Debian thread about the issue:
https://lists.debian.org/debian-python/2016/04/msg00074.html

**Comment by jtauber** [comment]
I don't think the tests should ship with the package normally. Is there a workaround I can help with?

**Comment by jtojnar** [comment]
I think you are supposed to include the unit test into the tarball but not into the wheel.

Looking through the [packaging docs](https://packaging.python.org/), I could not find anything mentioning whether to include tests into the tarball. The following StackOverflow answers suggest they are supposed to be included:

https://stackoverflow.com/a/16444257/160386
https://stackoverflow.com/a/32391295/160386
https://stackoverflow.com/a/14437754/160386

Out of [top 15 packages](https://hugovk.github.io/top-pypi-packages/) on PyPI, only three (pip, awscli and colorama) do not include tests in the tarballs so it seems to be common practice.

**Comment by jtauber** [comment]
yep, makes sense to include them in a source distribution. I just need to work out how to now :-)

**Comment by jtojnar** [comment]
Requests seem to add it through `MANIFEST.in`: https://github.com/requests/requests/blob/master/MANIFEST.in

**Comment by pgajdos** [comment]
I can second that. Unit tests can be used by packagers to test the distro package.
